README.md

```diff
@@ -1,11 +1,6 @@
 <a name="readme-top"></a>
```

Comment:
- [ ] There are multiple sections on your readme that need completion/customization:

1. 
![image](https://github.com/Bludidi/vehi-go-frontend/assets/57421823/36dd4dc5-c68f-45b9-97a6-104bab3a1544)

2.
![image](https://github.com/Bludidi/vehi-go-frontend/assets/57421823/2c603d90-999a-400b-8f7a-0edfd50fdb93)

3. This section is the most important.  It needs to explain how to run your project properly:
![image](https://github.com/Bludidi/vehi-go-frontend/assets/57421823/f9b4acbe-7a40-4f80-ba91-3c688f52098c)

4.
![image](https://github.com/Bludidi/vehi-go-frontend/assets/57421823/b5b9695a-cc3f-4693-8161-7e90d0492fb1)

5.
![image](https://github.com/Bludidi/vehi-go-frontend/assets/57421823/ab9f0162-5075-4b84-8e05-7d489d6a8014)


The only optional section is the `FAQ` you can either complete it or remove it.  Since all the other sections are required, kindly please fill them out with the relevant info for your project.

That way, your project will be well documented. 💪